examples/mnv1.cpp

```diff
@@ -0,0 +1,225 @@
+/**
```

Comment:
This example seems to have a lot in common with our existing MNIST example. Did you consider adding a flag to the old one, so that it would optionally create more complicated net architecture, using group convolution?

Having two very similar examples with shared code may be difficult to maintain in the long run.